clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/templates/AzureLoadBalancerResourceTemplate.groovy

```diff
@@ -95,7 +95,13 @@ class AzureLoadBalancerResourceTemplate {
       name = "[variables('loadBalancerName')]"
       type = "Microsoft.Network/loadBalancers"
       location = "[parameters('location')]"
-      tags = ["appName":description.appName, "stack":description.stack, "detail":description.detail]
+      tags = [:]
+      tags.appName = description.appName
+      tags.stack = description.stack
+      tags.detail = description.detail
+      if (description.cluster) tags.cluster = description.cluster
+      if (description.serverGroup) tags.serverGroup = description.serverGroup
+      if (description.vnet) tags.vnet = description.vnet
```

Comment:
Should we be doing this check whenever we set a tag? I realize the likelihood of some of the values being null might be low but template will validation will fail if a tag is ever set to empty or null